src/language-css/parser-postcss.js

```diff
@@ -53,6 +55,34 @@ function parseValueNode(valueNode, options) {
       node.unit = "...";
     }
 
+    // Work around postcss bug parsing `list-*` as the word `list-` and operator `*`.
+    // postcss-values-parser flagged this as a wontfix: https://github.com/shellscape/postcss-values-parser/issues/108
```

Comment:
We really need to migrate on own parser, `postcss-values-parser` is kitchen sink with a lot of bugs, spec violations and etc trash

Reply:
Agree long-term.